sources/br/sp/sao-paulo-city.json

```diff
@@ -0,0 +1,43 @@
+{
+    "schema": 2,
+    "coverage": {
+        "ISO 3166": {
+            "alpha2": "BR-SP",
+            "country": "Brazil",
+            "state": "São Paulo"
+        },
+        "country": "br",
+        "state": "sp",
+        "geometry": {
+            "type": "Point",
+            "coordinates": [
+                -46.63527,
+                -23.54551
+            ]
+        }
+    },
+    "layers": {
+        "addresses": [
+            {
+                "name": "GeoSampa",
+                "data": "https://github.com/geoinfo-smdu/georeferenciamento-do-iptu/raw/master/resultados/sao-paulo-address-IPTU-2021.csv.zip",
+                "website": "http://geosampa.prefeitura.sp.gov.br/",
+                "protocol": "http",
+                "conform": {
+                    "format": "csv",
+                    "srs": "EPSG:4326",
+                    "accuracy": 2,
+                    "number": "numero",
+                    "street": "logradouro",
+                    "unit": "complemento",
+                    "lon": "longitude",
+                    "lat": "latitude",
+                    "postcode": "cep",
+                    "id": "sqlc",
+                    "addrtype": "uso"
+                },
+                "compression": "zip"
+            }
+        ]
+    }
+}
```

Comment:
Can you add a newline at the end of the file please?

Reply:
Done